packages/components/src/components/pin-code/pin-code-styles.ts

```diff
@@ -45,11 +45,18 @@ export const getComponentCss = (
           opacity: 0.2, // TODO: not in sync with e.g. checkbox/radio-button loading style
           cursor: 'not-allowed',
         }),
-        ...Object.fromEntries(
+        // since @playwright/test@1.40.1 this does not work anymore in Webkit browser engine for unknown reasons
```

Comment:
Should we really change implementations to make tests work?

Reply:
Well, normally not but I really have no glue why it stopped working. It's also nearly impossible to debug since it only happens with Playwright Webkit browser engine but not with e.g. macOS Safari. 
On the other hand the new implementation might be a bit larger in size but is also more readable and since this component won't be heavily used I think it's okay.